src/mame/heathkit/h89.cpp

```diff
@@ -46,6 +46,7 @@
 #include "tlb.h"
 #include "z37_fdc.h"
 #include "intr_cntrl.h"
+#include "h_88_cass.h"
```

Comment:
Please keep the `#include` statements sorted within each section – they get unmanageable otherwise.